fsspec_reference_maker/hdf.py

```diff
@@ -71,14 +72,14 @@ class Hdf5ToZarr:
         Zarr array. Default is ``False``.
     """
 
-    def __init__(self, h5f, xarray=False):
+    def __init__(self, h5f: Union[str, BinaryIO], xarray: bool = False):
         # Open HDF5 file in read mode...
         lggr.debug(f'HDF5 file: {h5f}')
         lggr.debug(f'xarray: {xarray}')
         self._h5f = h5py.File(h5f, mode='r')
         self._xr = xarray
 
-        self.store = {}
+        self.store = zarr.MemoryStore()
```

Comment:
Why change this?

Reply:
Looked nicer, more explicit to me. Returned to the ordinary dict.